src/seedsigner/gui/components.py

```diff
@@ -28,7 +28,7 @@ class GUIConstants:
     REGTEST_COLOR = "#00caf1"
 
     ICON_FONT_NAME__FONT_AWESOME = "Font_Awesome_6_Free-Solid-900"
-    ICON_FONT_NAME__SEEDSIGNER = "seedsigner-glyphs"
+    ICON_FONT_NAME__SEEDSIGNER = "seedsigner-icons"
     ICON_FONT_SIZE = 22
     ICON_INLINE_FONT_SIZE = 24
     ICON_LARGE_BUTTON_SIZE = 36
```

Comment:
For your LargeButtonScreen where your latest icon set appears smaller than before, you might play with resizing this from 36 to something bigger.

I don't know what bad side-effects this might have.  @kdmukai will probably know better.

Reply:
Changed to 48 and it's fixed. I can't find anything that broke.
![MainMenuView](https://github.com/SeedSigner/seedsigner/assets/86452817/dce1a043-b53f-4a2e-ba8a-d5f6bde471be)